tests/UI/campaigns/functional/BO/02_orders/01_orders/04_checkInvoiceDownloadedFromList.js

```diff
@@ -234,7 +161,7 @@ describe('BO - orders : Check invoice downloaded from list', async () => {
       await expect(pageTitle, 'Fail to open order history page').to.contains(foOrderHistoryPage.pageTitle);
     });
 
-    it('should check last order status in FO', async function () {
+    it('should check last order status', async function () {
```

Comment:
Test looks related to FO only

Reply:
I delete 'in FO' because in the describe I mentioned that I will 'Check order status in FO' so no need to rewrite it on the step

Reply:
You did well indeed :)